internal/controller/handler-hook/handler_hook.go

```diff
@@ -18,6 +18,12 @@ func New(wrapped handler.EventHandler, hookFunc HookFunc) handler.EventHandler {
 		hookFunc: hookFunc,
 	}
 }
+func New1(wrapped handler.EventHandler, hookFunc HookFunc) handler.EventHandler {
```

Comment:
This is the exact same function as the one above.

Reply:
Ups, you are right! That was a test that I have forgotten to remove :sweat_smile:
I will remove it.

Reply:
Removed.